### PR TITLE
Update configuration for arrow

### PR DESCRIPTION
**Pull request by rachelcarmena**
# Pull request motivation(s)

Please, we'd like to update the configuration:

- Replace the current selector at level 0
- Index pages which filename is different to `index.html`

You can find me as a member here: https://github.com/orgs/arrow-kt/people

### What is the current behaviour?

- The current selector at level 0 isn't providing value.
- Pages which filename is different to `index.html` aren't being indexed (sorry @shortcuts for this new decision to remove it :pray: )

### What is the expected behaviour?

- Having `.doc-content h1` selector at level 0
- Index pages which filename is different to `index.html` will be indexed.

##### NB: Do you want to request a **feature** or report a **bug**?

It's a configuration improvement for a site.

##### NB2: Any other feedback / questions ?

Thank you so much for your support! :pray:

### CODE REVIEW

File: configs/arrow.json [hunk 1]
```diff
@@ -3,21 +3,15 @@
   "start_urls": ["https://arrow-kt.io"],
   "stop_urls": [
     "https://arrow-kt.io/docs/[0-9]",
-    "https://arrow-kt.io/docs/next",
-    ".*(?<!index\\.html)$"
```
Comment (shortcuts) [inline configs/arrow.json:7]: Is the goal of removing the `index.html` filter to list the API/functions? 

Adding `.doc-content tr > td:first-child` would do the same without adding duplicates

You'd have something like this

Screenshot 2021-04-05 at 15 03 18
 to the search results.

File: configs/arrow.json [hunk 2]
```diff
@@ -3,21 +3,15 @@
...
-      },
-      "lvl1": ".doc-content h1",
-      "lvl2": ".doc-content h2",
-      "lvl3": ".doc-content h3",
-      "lvl4": ".doc-content h4",
+      "lvl0": ".doc-content h1",
+      "lvl1": ".doc-content h2",
+      "lvl2": ".doc-content h3",
+      "lvl3": ".doc-content h4",
       "text": ".doc-content p, .doc-content li"
```
Comment (shortcuts) [inline configs/arrow.json:15]:
```suggestion
      "text": ".doc-content p, .doc-content li, .doc-content tr > td:first-child"
```

If what I said in the above comment suits you

File: configs/arrow.json [hunk 3]
```diff
@@ -3,21 +3,15 @@
...
   ],
   "sitemap_urls": ["https://arrow-kt.io/sitemap.xml"],
   "selectors": {
     "default": {
-      "lvl0": {
-        "selector": ".brand-title",
-        "global": true,
-        "default_value": "Documentation"
-      },
-      "lvl1": ".doc-content h1",
```
Comment (shortcuts) [inline configs/arrow.json:17]: Do you have any query examples where it doesn't work?

Replacing this selector would add a lot of noise to the results I think

|before|after|
|--|--|
|Screenshot 2021-04-05 at 15 09 33|Screenshot 2021-04-05 at 15 11 24|

File: configs/arrow.json (hunk 1 above)
Comment (rachelcarmena) [reply configs/arrow.json]: Thanks @shortcuts !! :raised_hands:

File: configs/arrow.json (hunk 3 above)
Comment (rachelcarmena) [reply configs/arrow.json]: Undone, thanks!!

**Comment by rachelcarmena** [comment]
I updated the configuration with your amazing feedback, @shortcuts, thank you so much!! :raised_hands:

**Comment by shortcuts** [comment]
> I updated the configuration with your amazing feedback, @shortcuts, thank you so much!! 🙌

🎉 Feel free to merge master and I'll merge the PR!

**Review by shortcuts — APPROVED** [review]

**Comment by rachelcarmena** [comment]
> :tada: Feel free to merge master and I'll merge the PR!

Thank you so much Clément, done!! :raised_hands:

**Comment by rachelcarmena** [comment]
Really happy with the results @shortcuts :raised_hands: Thanks again!!

**Comment by shortcuts** [comment]
Glad to hear that @rachelcarmena :D enjoy!

**Pull request merged**